### DISCUSSION TITLE
There is a coding error by other programs that I need to bypass.

### DISCUSSION THREAD

**Discussion by Phuxxi** [cboxdoerfer/fsearch#518] [General]
Hi all this is a simple problem that involves several parties.

First off Christian Boxdoerfer makes a truly exceptional search program for Linux called Fsearch.

Ryo Nakano   Reco  -  Record talks to remember the contents later

https://github.com/ryonakano/reco

Take photos and record videos with your webcam

https://webcamoid.github.io/

The problem I am now facing is that when I do a search for files - mp3's, pictures, videos, documents etc., using Fsearch, these programs that have NOTHING to do with any thing, have backed up a file NAME list - of everything on my hard drives... like 1.5 million files.

And Fsearch cannot be faulted, but after much looking I can't see any overt ways to cut specific directories from the search.

You see there are legitimate file name pathways, e.g. Home/Desktop/Elvis Movies/ etc. And then there are these files... 

Mostly these two: And there are copies of EVERY file NAME on the system....

/run/user/1000/doc/by-app/com.github.ryonakano.reco/ etc.

/run/user/1000/doc/by-app/io.github.webcamoid.Webcamoid/ etc.

So the programs have coding defects.

Can the people responsible for the apparently copied or replicated coding error please clean them up.

And Christian - if you could include a way to permanently lock out some pathways when doing a search.

Thanks to all.

Shane T. Hanson

![Idi Amin](https://github.com/cboxdoerfer/fsearch/assets/72900099/338110e0-04b7-48f0-a584-cbfde4b29f47)

**Comment by cboxdoerfer** [comment]
Hi Shane,

you can exclude the `/run/user/1000/doc` directory in the database settings (_Preferences -> Database -> Exclude_). This way those unwanted results won't show up.
